### ISSUE TITLE
Country Selection Improvements

### ISSUE THREAD

**Issue by brunobowden** [WorldHealthOrganization/app#1824]
## Summary

When the user selects their home country. The app should make it easier to do this.

## Desired Solution

1. Float likely countries to the top of the list (e.g. based on IP Geolocation)
2. Add country flags for visual recognition and to accomodate people with limited reading ability or lack of translation to their local language

## Checklist:

- [x] Searched the [existing issues](https://github.com/WorldHealthOrganization/app/issues) to ensure you are not creating a duplicate.
- [x] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md).
